packages/core/src/utils/debug.test.ts

```diff
@@ -0,0 +1,57 @@
+import {describe, test, beforeEach, expect} from 'vitest';
+import {setProject, useLogger, debug} from '../utils';
+import {Project, ProjectMetadata} from '../Project';
+import {Meta} from '../Meta';
+import {LogLevel, LogPayload} from '../Logger';
+import {Rect, Vector2} from '../types';
+
+describe('debug()', () => {
+  beforeEach(() => {
+    setProject(
+      new Project('test', new Meta<ProjectMetadata>('test'), {scenes: []}),
```

Comment:
This is a deprecated signature, we should pass a config object instead:
```suggestion
      setProject(new Project({name: 'test', scenes: []}));
```
Also, metadata is not necessary.

Reply:
Oh, weird. VSCode was telling me that _that's_ the deprecated signature. Also the deprecation notice in `Project.ts` seems to agree with that, too: https://github.com/motion-canvas/motion-canvas/blob/main/packages/core/src/Project.ts#L190

Reply:
Right, I most likely forgot that each signature should have its own typedoc and thought I was deprecating all of them.
But the one with the config object is the correct one.